vue/components/organisms/configurator/configurator.vue

```diff
@@ -235,6 +235,12 @@ export const Configurator = {
         }
     },
     computed: {
+        loadingCsrConfigurator() {
+            if (!this.ripeInstance.loadedConfig) return false;

```

Comment:
I think this should be the bottow one, as the other ones are immediate returns

Reply:
I'm going to omit this one since `this.ripeInstance.hasStrategy("csr")` also checks for this